1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,31 @@
 function highlightWords(paragraph, colours) {
   // Write your code here...
+  let content = document.getElementById("content");
+  let para = document.createElement("p");
+  content.appendChild(para);
+  let select = document.createElement("select");
+  content.appendChild(select);
+
+  ["Colours", ...colours].forEach((colour) => {
+    let option = document.createElement("option");
+    option.innerText = colour;
+    select.appendChild(option);
+  });
+
+  paragraph.split(" ").forEach((word) => {
+    let span = document.createElement("span");
+    span.innerText = word + " ";
+    para.appendChild(span);
+
+    span.addEventListener("click", () => {
+      let selected = select.value;
+      if (selected === "none") {
```

Comment:
I don't think 'none' is one of the values in the select dropdown? The default is 'Colours'. So if you click on a word that has a background colour, it's not removing the colour style.